src/db/config.ts

```diff
@@ -30,6 +30,7 @@ export const getDBConfig = (configService: ConfigService) => {
         'DB_CONNECTION_TIMEOUT',
         1000
       ),
+      idleTimeoutMillis: 10000,
```

Comment:
@whitneypurdum sorry, I was confused in our previous discussion. I think it's better to adjust `connectionTimeoutMillis`, rather than `idleTimeoutMillis`